### DISCUSSION TITLE
Inline hints not working

### DISCUSSION THREAD

**Discussion by dkkopp** [apache/netbeans#5570] [Q&A] [answered]
I have turned on all three inline hints, but nothing shows up in my editor.

I have tried this in NetBeans 17 on both Ubuntu and MaoOS, both running on Azul Zulu 19.

Anyone else having this problem?

**Comment by mbien** [comment, marked as answer]
is `view -> show inline hints` also enabled?

**Reply by dkkopp** [reply]
Oops. Thanks.
